facturacion_mexico/ereceipts/tests/test_ereceipts_fase0.py

```
@@ -0,0 +1,430 @@
+	"""Tests para el payload enviado a FacturAPI al crear E-Receipt."""
+
+	def _make_mock_si_item(self, item_code, qty=1, rate=100, uom="H87 - Pieza"):
+		item = MagicMock()
+		item.item_code = item_code
+		item.item_name = "Producto Test"
+		item.qty = qty
+		item.rate = rate
+		item.uom = uom
+		return item
+
+	def _make_mock_si(self, customer="CUST-001", grand_total=116):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_make_mock_si`

(ANN202)